### DISCUSSION TITLE
Has Many Attach Existing Records Not Showing

### DISCUSSION THREAD

**Discussion by NigelDorning** [filamentphp/filament#358] [Help]
Hi,

I've set up a relation manager with a hasMany relationship but I can't see anyway of attaching an existing record. I have seen this working when I was playing around with another project, but can't seem to get it working again.

Thanks again for your hard work.

**Comment by lironsam** [comment]
Yes, I have the same question. 
I have set up a relationManager with hasMany and when I would like to attach a specific model I have only the create method.

For example, I have model A that can have many model B. When I select a specific model A, I can see some model B attributed but I cannot attach new one directly.

thank

**Reply by danharrin** [reply]
We haven't implemented the Attach button on has-many managers yet, but this will come at some point. It's already released for belong-to-many managers.

**Comment by crystaldaking** [comment]
any updates?
